app/views/companies/_form.html.haml

```diff
@@ -20,12 +21,19 @@
   = f.label :website, "#{t('companies.website_include_http')}"
   = f.text_field :website, class: 'wpcf7-form-control'
 
-  = f.label :dinkurs_company_id, "#{t('companies.show.dinkurs_key')}"
+  .dinkurs_company_id
+    = f.label :dinkurs_company_id, "#{t('companies.show.dinkurs_key')}"
 
-  %span.glyphicon.glyphicon-info-sign{ title: "#{t('.dinkurs_id_tooltip')}",
-                                       data: {toggle: 'tooltip'} }
+    %span.glyphicon.glyphicon-info-sign{ title: "#{t('.dinkurs_id_tooltip')}",
+                                         data: {toggle: 'tooltip'} }
 
-  = f.text_field :dinkurs_company_id, class: 'wpcf7-form-control'
+    = f.text_field :dinkurs_company_id, class: 'wpcf7-form-control'
+
+  .dinkurs_company_show
+    = f.label :show_dinkurs_events, "#{t('companies.show_dinkurs_events')}"
```

Comment:
We need a tooltip for the switch - so that a user knows what it does and how to set it to on (show events) and off (hide events).  In English it could something like this:
```
Set this switch to on (toggle to the right) to show upcoming Dinkurs events on your company page.
```